tests.py

```diff
@@ -0,0 +1,114 @@
+"""
+Validate log configuration
+"""
+import logging
+import os
+import unittest
+
+import flask
+# pylint: disable=no-name-in-module,import-error
+from flask.ext.log import Logging
```

Comment:
pylint doesn't handle dynamic imports well.  https://bitbucket.org/logilab/pylint/issue/413/add-dynamic-modules-option-to-allow-for and https://bitbucket.org/logilab/pylint/issue/58/ are part of the reason why.  I think they fixed it for specific modules, but the magic flask extension import isn't covered as far as I know